### PR TITLE
Fix nodes' label for ROSA clusters

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://bugzilla.redhat.com/show_bug.cgi?id=2302235

### CODE REVIEW

File: packages/shared/src/constants/common.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const DASH = '-';
 export const AVAILABLE = 'Available';
-export const CEPH_STORAGE_NAMESPACE = 'openshift-storage';
+export const DEFAULT_STORAGE_NAMESPACE = 'openshift-storage'; // This hardcoded namespace is only for specific cases, do not use it otherwise.
```
Comment (bipuladh) [inline packages/shared/src/constants/common.ts:3]: This is going to cause confusion just use this value directly in that function. Else people are going to use it in multiple places.

Comment (SanjalKatiyar) [reply packages/shared/src/constants/common.ts:3]: we anyway need it here: https://github.com/red-hat-storage/odf-console/blob/master/packages/odf/components/alerts/alert-action-path.tsx#L24, so I can't remove this variable entirely... and using directly in function would mean to repeating `openshift-storage` at multiple places (had it been only single or twice instances, I would have opted for direct value)...

Comment (SanjalKatiyar) [reply packages/shared/src/constants/common.ts:3]: now we are only keeping `openshift-storage` at a single place...

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx [hunk 2]
```diff
@@ -251,7 +250,7 @@ export const AddCapacityModal: React.FC<AddCapacityModalProps> = ({
   const replica = getDeviceSetReplica(
     isArbiterEnabled,
     hasFlexibleScaling,
-    createWizardNodeState(getCephNodes(nodesData, odfNamespace) as NodeData[])
+    createWizardNodeState(getCephNodes(nodesData, ocsClusterNs) as NodeData[])
```
Comment (bipuladh) [inline packages/odf/modals/add-capacity/add-capacity-modal.tsx:253]: Just use the variable directly here. Instead of relying on `ocsClusterNs`

Comment (SanjalKatiyar) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:253]: `getCephNodes` will work even if I pass empty string here, I passed `ocsClusterNs` to keep it aligned with dashboards and other places.

Originally (in 4.15, when label was dynamic) I should have passed cluster namespace here (i.e., ocsClusterNs), but I missed earlier.

Anyway, `getCephNodes`, `cephStorageLabel` & `labelNodes` will internally use hardcoded `openshift-storage`... and when we support multiple internal clusters we can just update these functions and all other places (where these functions are called will automatically be fixed).
I have added related comments on top of these functions as well.

Comment (SanjalKatiyar) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:253]: Idea is to not update the code where these functions are called (as much as possible), rather update/hack the functions themselves.

Other way is: I go to each file and pass `DEFAULT_STORAGE_NAMESPACE` to the function and do not update functions' code itself... but that probably will be more number of changes than the current fix and will need more updates later when/if we support multiple internal clusters.





[4:13](https://ibm-systems-storage.slack.com/archives/D04SUE8L3HS/p1722940995343479)
I can do this other way as well, if that make more sense... but probably that will be slightly more number of changes than current fi

Comment (SanjalKatiyar) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:253]: Let me know which one you prefer.

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**